src/pages/GovernContent.jsx

```diff
@@ -245,7 +247,7 @@ export default function Govern() {
             <h3>Algorand Governance Period #5</h3>
             <div style={{ fontSize: 11 }}>Registration Ends</div>
             <CountDownContainer>
-              <CountdownTimer targetDate={commitmentPeriodEnd} showZero={new Date().getTime() > commitmentPeriodEnd} />
+              <CountdownTimer targetDate={commitmentPeriodEnd} showZero={false} />
```

Comment:
I think the all-zero looks better than the built-in enrollment ended message. Could we keep this as-is?